core/_neat.scss

```diff
@@ -0,0 +1,11 @@
+// Neat 2.0.0.alpha.0
+// http://neat.bourbon.io
+// Copyright 2015 thoughtbot, inc.
+// MIT License
+
+@charset "UTF-8";
```

Comment:
`@charset` must be the first element, per https://developer.mozilla.org/en-US/docs/Web/CSS/@charset